arrows/vxl/image_io.cxx

```diff
@@ -164,12 +275,75 @@ class image_io::priv
                          this->intensity_range);
   }
 
+  template <typename pix_t>
```

Comment:
pedantic: should have spaces inside `<>`s and no space in `template<` (multiple instances)